joomla/joomla-cms
# Fixing sql fields

Pull Request for Issue #13088.

### Summary of Changes
* Adds PostgreSQL commands that were missing in previous PR
* Merges update commands related to com_fields into one file.

### Testing Instructions
* Test an update from 3.6.4 to staging with MySQL and/or PostgreSQL and verify that fields and firleds group managing works.
* Also test a fresh install of staging

### Documentation Changes Required
None

#### Notes
Once this is merged, fields still need SQL Azure SQL commands.
